<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>company</class>
 <widget class="QDialog" name="company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="updateCompanyInfoPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>413</width>
     <height>337</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="14" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2" colspan="2">
     <widget class="QLineEdit" name="websiteLineEdit"/>
    </item>
    <item row="12" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Website</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>GSTIN</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2" colspan="2">
     <widget class="QLineEdit" name="panLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>District</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>State</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="countryLineEdit"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>EMail</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="pincodeLineEdit"/>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>PAN</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="stateComboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QLineEdit" name="cinLineEdit"/>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Taxation Type</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2" colspan="2">
     <widget class="QLineEdit" name="gstinLineEdit"/>
    </item>
    <item row="16" column="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Contact 2</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Contact 1</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QComboBox" name="taxationtypeComboBox"/>
    </item>
    <item row="17" column="1" colspan="2">
     <widget class="QLineEdit" name="contact1LineEdit"/>
    </item>
    <item row="17" column="3">
     <widget class="QLineEdit" name="contact2LineEdit"/>
    </item>
    <item row="11" column="0">
     <widget class="QLineEdit" name="cityLineEdit"/>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pin Code</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="districtLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
